SingularityService/src/main/java/com/hubspot/singularity/resources/RequestResource.java

```diff
@@ -149,7 +149,11 @@ public SingularityRequestParent submit(@ApiParam("The Singularity request to cre
 
     final long now = System.currentTimeMillis();
 
-    requestManager.activate(newRequest, maybeOldRequest.isPresent() ? RequestHistoryType.UPDATED : RequestHistoryType.CREATED, now, JavaUtils.getUserEmail(user));
+    if (maybeOldRequestWithState.isPresent() && maybeOldRequestWithState.get().getRequest().isLongRunning()) {
```

Comment:
if i'm reading this right, paused cron jobs will still get un-paused. i'd welcome @wsorenson's 2 cents on this, but i think we should only call `activate()` if the request didn't previously exist